client/me/billing-history/transactions-header.jsx

```diff
@@ -179,16 +179,16 @@ class TransactionsHeader extends React.Component {
 	renderAppsPopover() {
 		const { appFilters, filter, translate } = this.props;
 		const selectedFilter = find( appFilters, { value: filter.app } );
-		const selectedText = selectedFilter ? selectedFilter.title : translate( 'All Apps' );
+		const selectedText = selectedFilter ? selectedFilter.title : translate( 'All apps' );
 
 		return (
 			<SelectDropdown
 				selectedText={ selectedText }
 				onClick={ this.handleAppsPopoverLinkClick }
 				className="billing-history__transactions-header-select-dropdown"
 			>
-				<SelectDropdown.Label>{ translate( 'App Name' ) }</SelectDropdown.Label>
-				{ this.renderAppPicker( translate( 'All Apps' ), 'all' ) }
+				<SelectDropdown.Label>{ translate( 'App name' ) }</SelectDropdown.Label>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.